integration_tests/tests/acm.rs

```diff
@@ -13,5 +13,5 @@ fn should_list_certificates() {
     let client = AcmClient::new(default_tls_client().unwrap(), credentials, Region::UsEast1);
     let request = ListCertificatesRequest::default();
 
-    client.list_certificates(&request).unwrap();
+    client.list_certificates(&request).sync().unwrap();
```

Comment:
I'd be a lot more work afaics to make the integration tests async. I'd recommend postponing this until generator functions land in stable and work well with futures.

Because `.sync()` technically relies on the underlying async mechanisms, they should already be covered, but I can add a couple of "async" tests here and there.